packages/bundle-source/src/transform.js

```diff
@@ -0,0 +1,121 @@
+import * as babelParser from '@babel/parser';
```

Comment:
fyi @naugtur, I factored this transform out in preparation for it to move to `@endo/transforms`.